src/SUMMARY.md

```diff
@@ -34,7 +34,7 @@
 	* [Destructors](destructors.md)
 	* [Leaking](leaking.md)
 * [Unwinding](unwinding.md)
-	* [Exception Safety](exception-safety.md)
+	* [例外安全性](exception-safety.md)
```

Comment:
(I'm not super familiar with the "pull request" system in github, but...) can you exclude this change from this PR?

Reply:
I can, but I think that's not a good idea. Usually PR is not for changing only one file, but for "chunk of changes of one purpose." Splitting up this change to another PR will be too fine grained.

Reply:
I understand that. My point was this PR is about poisoning and here "Exception Safety" is translated, which seems odd.
not a big deal. I'm going to merge this change.
thx a lot!!!